README.md

```diff
@@ -355,6 +362,75 @@ property in your domain data, this data will be reflect automatically in the Dat
 be able to see the data through the Data Index anymore, although the data still persisted in Infinispan._
 
+## Kogito Jobs Service deployment
+
+Like Data Index, [Jobs Service](https://github.com/kiegroup/kogito-runtimes/wiki/Jobs-Service) can be deployed via Operator or CLI. 
+If persistence is required, the operator will also deploy an Infinispan server using Infinispan Operator.
+
+### Kogito Jobs Service installation
+
+There's a couple of ways to install the Jobs Service into your namespace using Kogito Operator.
+
+#### Installing the Kogito Jobs Service with the Kogito CLI
+
+If you have installed the [Kogito CLI](https://github.com/kiegroup/kogito-cloud-operator#kogito-cli), 
+run the following command to create the Kogito Jobs Service resource:
+
+``` 
+$ kogito install jobs-service -p my-project
+```
+
+There's some options to customize the Jobs Service deployment with CLI. 
+Run `kogito install jobs-service --help` to understand and set them according to your requirements.  
+
+#### Installing the Kogito Jobs Service with the Operator Catalog (OLM)
+
+If you are running on OpenShift 4.x, you can use the OperatorHub user interface to create the Kogito Jobs Service resource. 
+In the OpenShift Web Console, go to **Installed Operators** -> **Kogito Operator** -> **Kogito Jobs Service**. 
+Click **Create Kogito Jobs Service** and create a new resource as shown in the following example:
+
+```ỳaml
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoJobsService
+metadata:
+  name: jobs-service
+spec:
+  replicas: 1
+```
+
+#### Installing the Kogito Data Index Service with the oc client
+
+To create the Kogito Jobs Service resource using the oc client, you can use the CR file from the previous example as a reference 
+and create the custom resource from the command line as shown in the following example:
+
+```bash
+ # Clone this repository
+ $ git clone https://github.com/kiegroup/kogito-cloud-operator.git
+ $ cd kogito-cloud-operator
+ # Make your changes
+ $ vi deploy/crds/app.kiegroup.org_v1alpha1_kogitojobsservice_cr.yaml
+ # Deploy to the cluster
+ $ oc create -f deploy/crds/app.kiegroup.org_v1alpha1_kogitojobsservice_cr.yaml -n my-project
+```
+
+### Enabling Persistence with Infinispan
+
+Jobs Service supports persistence with Infinispan by setting the property `spec.infinispan.useKogitoInfra` to `true` in the
+CR or the flag `--enable-persistence` in the CLI.
+
+When doing this Kogito Operator deploys a new Infinispan server via Infinispan Operator for you within the same namespace
+that you're deploying Jobs Service. It also sets all information regarding server authentication.
+
+For this to work, bear in mind that Infinispan Operator must be installed in the namespace. If the Kogito Operator was installed
+via OLM, changes are that the Infinispan Operator would also be installed. If installed manually, you will also have to manually
```

Comment:
the "changes are that the Infinispan Operator would also be installed" is not clear to me. maybe rephrase?